blog/models.py

```diff
@@ -13,6 +13,11 @@ def __str__(self):
     def get_absolute_url(self):
         return reverse('home')
 
+    def save(self, *args, **kwargs):
+        superuser_id = User.objects.filter(is_superuser=True).values('id')
```

Comment:
This logic will not work in case when you have more than one superuser

Reply:
done